engine/apps/mobile_app/urls.py

```diff
@@ -34,11 +33,10 @@
     optional_slash_path("fcm_relay", FCMRelayView.as_view(), name="fcm_relay"),
 ]
 
-if settings.MOBILE_APP_GATEWAY_ENABLED:
```

Comment:
this check is now [here](https://github.com/grafana/oncall/pull/3518/files#diff-cd8caa0fd8e0fba883880308df6d0dfc5db91323ab90207bf155d97fc810fcf3R104-R105)